### PR TITLE
[stable/sentry] add new configurable features

### PR DESCRIPTION
**Pull request by mrgleeco**
This is an updated/revised version of https://github.com/kubernetes/charts/pull/2737

### CODE REVIEW

**Comment by mrgleeco** [comment]
fyi @unguiculus @rothgar

File: stable/sentry/values.yaml [hunk 1]
```diff
@@ -110,15 +114,43 @@ ingress:
   #     hosts:
   #       - sentry.local
 
+  ## or, if using tlsSecret, enumerate the cert hostnames here
+  tlsHosts: []
+
+## tlsSecret allows a cert to be used for ingress
+#  consider putting this w/ other secrets in seperate encrypted values file
+# tlsSecret:
+#    key:  |
```
Comment (jody-frankowski) [inline stable/sentry/values.yaml:123]: This is clearly a bad practice (as most user will git their values.yaml), and I think that we shouldn't suggest such solutions to the user.

File: stable/sentry/values.yaml [hunk 2]
```diff
@@ -110,15 +114,43 @@ ingress:
...
+#    crt: |
+#    ...
+
 # TODO: add support for plugins https://docs.sentry.io/server/plugins/
 
+## allow these chart requirements to be omitted
+## and replaced with external services (using remoteHost)
+requiredBackend:
+    postgresql: true
+    redis: true
```
Comment (jody-frankowski) [inline stable/sentry/values.yaml:136]: Isn't it easier to just check if `redis.remote` is set? By the way the README hasn't been updated in accordance.

**Pull request closed without merging**